include/envoy/filter/http/BUILD

```diff
@@ -0,0 +1,21 @@
+load(
```

Comment:
nit: sorry to be a pain by why make a new filter directory here? Why not just put all of this directly in include/envoy/http? I don't feel that strongly about it if you do. Same below.

Reply:
This was requested by @htuch . I previously placed it into router directory. I don't have strong opinions on this, as long as I don't have to move it back and forth.

Reply:
If you already discussed with @htuch this is fine.